RIOT-OS/RIOT
# tests/littlefs: Move from unittests to regular test

### Contribution description

another one related to #10187

### Testing procedure

Verify that `tests/pkg_littlefs` works: `BOARD=<your-favourite> make -C tests/pkg_littlefs flash test`

### Issues/PRs references

#10187
